### PR TITLE
Translate menu action

### LINKED ISSUE #179: Context menu entry not translated
menu

Context menu entry is not translated.

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #179

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,6 @@
   },
   "dependencies": {
     "brace": "^0.11.1",
-    "marked": "^0.6.2"
+    "marked": "^0.7.0"
```
Comment (juliushaertl) [inline package.json]: done separately in #170

Comment (skjnldsv) [reply package.json]: Yeah, there was some audit issues, I decided to fix while at it :)

**Comment by skjnldsv** [comment]
> /compile /js

```
95 | modified: build/editor.js
96 | modified: build/editor.js.map
97 | modified: build/public-share.js
98 | modified: build/public-share.js.map
```

**Pull request merged**